task_1.py

```diff
@@ -0,0 +1,37 @@
+phone_book = {}
+
+while True:
+    user_data = input("Enter data(use command :add,delete,stats,show,list):")
+    if user_data == "last":
+        break
+
+    data = user_data.split()
+    command = data[0]
+    name = data[1] if len(data) > 1 else None
+
+    if command == "add":
+        phone = data[2]
+        if name and name not in phone_book:
+            phone_book[name] = phone
+            try:
+                value = int(phone)
+            except ValueError:
+                print("The phone number can only contain numbers")
+    elif command == "delete":
+        try:
+            value = phone_book[name]
+        except KeyError:
+            print("This user is not in the phone book")
+        if name and name in phone_book:
+            del phone_book[name]
```

Comment:
nit: could be simplified to:
```suggestion
        try:
            del phone_book[name]
        except KeyError:
            print("This user is not in the phone book")
```